src/components/Card/styled.ts

```diff
@@ -2,7 +2,7 @@ import Button from '@/components/Button';
 import Image from '@/components/Image';
 import { styled } from '@/styles';
 
-export const CARD_WIDTH = 248;
+export const CARD_WIDTH = 246;
```

Comment:
the border (`border: '1px solid #f1f1f1',`) of the container is 2 additional pixels to the width. 
If we want it to be rendered as 248, subtract 2.

This also allows us to remove https://github.com/voiceflow/react-chat/pull/6/files#diff-645fb0c711ca2ee4626dab7ba852ddd83a7e12848b71746546c89a0759d4a163L64-L68